File: teneasyChatSDK_Android/src/main/java/com/teneasy/sdk/ui/MessageItem.java
package com.teneasy.sdk.ui;

import java.io.Serializable;

public class MessageItem implements Serializable {

    private boolean isSend;
    private String msg;
    private long payLoadId;
    private String time;

    public MessageItem(boolean isSend, String msg, long payLoadId, String time) {
        this.isSend = isSend;
        this.msg = msg;
        this.payLoadId = payLoadId;
        this.time = time;
    }

    public boolean isSend() {
        return isSend;
    }

    public void setSend(boolean send) {
        isSend = send;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public long getPayLoadId() {
        return payLoadId;
    }

    public void setPayLoadId(long payLoadId) {
        this.payLoadId = payLoadId;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
}

File: teneasyChatSDK_Android/src/main/java/com/teneasy/sdk/ui/MessageItem.kt
package com.teneasy.sdk.ui

import api.common.CMessage
import java.io.Serializable

//enum MessageSendState: String { case 发送中="0", 发送成功="1", 发送失败="2", 未知="-1" }
enum class MessageSendState {
        发送中, 发送成功, 发送失败, 未知
}

class MessageItem {//:Serializable
        var id: Long = 0
        var isSend : Boolean = false
        //var sendError: Boolean = false
        var cMsg: CMessage.Message? = null
        var sendStatus: MessageSendState = MessageSendState.发送中
        //使用payLoadId来进行发送消息，消息回执的匹配
        var payLoadId: Long = 0
        var msgId: Long = 0
        var imgPath: String = ""
        //var status : Int = 0
    }